Adds generator generator docs (#793)
redwoodjs/redwoodjs.com: docs/cliCommands.md

17:21:49 api | ► http://localhost:8911/user/
```

### generate generator

> Copies a given generators template files to your local app for customization. The next time you generate that type again, it will use your custom template instead of Redwood's default.

Reviewer: - add apostrophe
```suggestion
Copies a given generator's template files to your local app for customization. The next time you generate that type again, it will use your custom template instead of Redwood's default.
```

```
yarn rw generate generator <name>
```

| Arguments & Options | Description                                                   |
| :------------------ | :------------------------------------------------------------ |
| `name`              | Name of the generator template(s) to copy (see help for list) |
| `--force, -f`       | Overwrite existing copied template files                      |

**Usage**

If you wanted to customize the page generator template, run the command:

```
yarn rw generate generator page
```

> And then check `web/generators/page` for the page, storybook and test template files. You don't need to keep all of these templates, you could customize just `page.tsx.template` and delete the others and they would still be generated, but using the default Redwood templates.

Reviewer: - feel like a semicolon or em dash is better here
```suggestion
And then check `web/generators/page` for the page, storybook and test template files. You don't need to keep all of these templates—you could customize just `page.tsx.template` and delete the others and they would still be generated, but using the default Redwood templates.
```

Author: ![image](https://user-images.githubusercontent.com/300/133840625-262314f6-7695-4419-ba94-5bba337d4048.png)

> The only exception to this rule is the scaffold templates. You'll get four directories, `assets`, `components`, `layouts` and `pages`. If you want to customize any one of the templates in those directories, you will need to keep all the other files inside of that same directory, even if you make no changes. (This is due to the way the scaffold looks up its template files.) For exmaple, if you wanted to customize only the index page of the scaffold (the one that lists all available records in the database) you would edit `web/generators/scaffold/pages/NamesPage.tsx.template` and keep the other pages in that directory. You _could_ delete the other three directories (`assets`, 'components`, `layouts`) if you just want to use the default templates for those.

Reviewer: - typos
```suggestion
The only exception to this rule is the scaffold templates. You'll get four directories, `assets`, `components`, `layouts` and `pages`. If you want to customize any one of the templates in those directories, you will need to keep all the other files inside of that same directory, even if you make no changes. (This is due to the way the scaffold looks up its template files.) For example, if you wanted to customize only the index page of the scaffold (the one that lists all available records in the database) you would edit `web/generators/scaffold/pages/NamesPage.tsx.template` and keep the other pages in that directory. You _could_ delete the other three directories (`assets`, `components`, `layouts`) if you just want to use the default templates for those.
```
